db/schemas/schemas.py

```diff
@@ -15,3 +17,16 @@ class Meta:
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+class JustScoreMetadata(SQLAlchemyAutoSchema):
+    """JustScoreMetadata The marshmallow class used to turn SQLAlchemy
+    rows into json for return in http responses."""
+
+    class Meta:
+        model = JustScore
+        include_relationships = True
+        load_instance = True
+    
+    application_id = auto_field(dump_only=True)
```

Comment:
⛏️ You can include foreign keys by setting a [parameter in the Meta. ](https://marshmallow-sqlalchemy.readthedocs.io/en/latest/api_reference.html#marshmallow_sqlalchemy.SQLAlchemyAutoSchemaOpts)